src/App.jsx

```diff
@@ -14,45 +16,126 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
+const SORT_BY_INDEX_DESC = 'reverse default';
+const ASC = 'asc';
+const DESC = 'desc';
+
+function getSortGoods(goods, sortField, sortDirection) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SORT_BY_ALPHABET:
+          return sortDirection === ASC
+            ? good1.localeCompare(good2)
+            : good2.localeCompare(good1);
+
+        case SORT_BY_LENGTH:
+          // eslint-disable-next-line no-nested-ternary
+          return sortDirection === ASC
```

Comment:
Nested ternary operators can make your code more difficult to understand and maintain. Please consider refactoring this part of your code.

Reply:
How to fix it exactly?

Reply:
Consider using if blocks or some different logic instead of nested ternary